openfl-tutorials/interactive_api/HPU/PyTorch_Kvasir_UNet/envoy/envoy_config_no_gpu.yaml

```diff
@@ -0,0 +1,12 @@
+params:
+  cuda_devices: []
+
+optional_plugin_components: {}
+
+shard_descriptor:
+  template: kvasir_shard_descriptor.KvasirShardDescriptor
+  params:
+    data_folder: kvasir_data
+    rank_worldsize: 2,10
```

Comment:
Please ensure the `rank` and `worldsize` correspond to actual number of nodes being run in the example. 
If this is intended to be a 2-collaborator example, for instance, `worldsize` should be 2

Reply:
Currently intended to be 1 collaborator example, so updated `worldsize` to 1